src/app/service/vote/Vote.ts

```diff
@@ -266,6 +272,25 @@ const fetchUser = async (clientAccountId) => {
 	}
 };
 
+// FIXME we will change this to openapi client in the future so we won't have to specify endpoints manually
+export const fetchGovernatorUser = async (governatorUserId) => {
+	const userGetEndpoint = `${process.env.GOVERNATOR_API_BASE_PATH}/${process.env.GOVERNATOR_API_PREFIX}/user/${governatorUserId}`;
```

Comment:
Maybe time to create a "getBaseEndpoint()" util

Reply:
Added a util, we can expand that to hold all the api call methods